spec/controllers/omniauth_callbacks_controller_spec.rb

```diff
@@ -0,0 +1,101 @@
+require 'rails_helper'
+
+RSpec.describe OmniauthCallbacksController, type: :controller do
+  describe '#github' do
+    context 'when github email doesn\'t exist' do
+      before(:each) do
+        github_omniauth
+        get :github
+        @user = User.find_by(email: 'user@email.com')
+      end
+
+      it 'expects omniauth.auth to be be_truthy' do
+        expect(request.env['omniauth.auth']).to be_truthy
+      end
```

Comment:
Try not to duplicate the description by the code. If you have no idea how to avoid it, you can just go with:
```ruby
it { expect(request.env['omniauth.auth']).to be_truthy }
```